projects/ng-keyboard-shortcuts/src/lib/keys.ts

```diff
@@ -1,4 +1,7 @@
-const isMac = (navigator.userAgent.indexOf('Mac OS') !== -1);
+let isMac = false;
+if (typeof navigator !== 'undefined') {
+    (navigator.userAgent.indexOf('Mac OS') !== -1);
```

Comment:
never set isMac to true in the iff
```suggestion
const  isMac = typeof navigation !== 'undefined' ? (navigator.userAgent.includes("Mac OS") : false;
```
though it still raises the question why we decide it is not Mac because navigator is not available at the current stage.

Reply:
Also I think we use new Typescript which supports:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html
which might makes the code a bit cleaner. just a suggestion, feel free to ignore.

Reply:
I applied your first suggestion